### DISCUSSION TITLE
[BUG] Docker - using a farmer address which we don't have the private keys for. We searched the first 500 addresses.

### DISCUSSION THREAD

**Discussion by evilight** [Chia-Network/chia-blockchain#5135] [Support]
**Describe the bug**
First, I use windows version of chia to start everything. I added a plots ( not finish yet ), but I found it is slow.
then, I tried to run it in docker on a Linux environment.
so, I copied c:\Users\user1\.chia to my Linux box.
then I use docker to run it.
docker run -v /root:/root -v /key:/key -e keys="/key/words.txt" -t 1f3983bd2694 /bin/bash

**To Reproduce**
1. install chia in Windows 10
2. generate key
3. start to add plots in Windows
4. copied c:\Users\user1\.chia to Linux /root (add plots not finished yet)
5. keep the 24 mnemonic seed in a words.txt
6. docker run -v /root:/root -v /key:/key -e keys="/key/words.txt" -t 1f3983bd2694 /bin/bash

**Expected behavior**
I expect the .chia from windows works perfect on Linux

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Linux]
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**

**Comment by gromran** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Architecture

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
